src/style.css

```diff
@@ -1 +1,96 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+:root {
+  --primary-color: #000;
+  --accent-color: #00acdc;
+  --background-color: #fff;
+  --header-line-height: 60px;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.container {
+  margin-inline: auto;
+  padding-inline: 50px;
+  max-width: 1200px;
+
+  @media (min-width: 1024px) {
+    max-width: 1024px;
+  }
+
+  @media (min-width: 1200px) {
+    max-width: 1200px;
+  }
+}
```

Comment:
Yes, it will work in css-preprocessors like `SASS`. But you are in `css` now, so there is no hierarchy included.
In this case, your media rules should be separate. Please update this part with appropriate modifications.

Reply:
Ths :blush: Really mixed up. Fixed.  All test shows[ Ok](https://adv555.github.io/layout_moyo-header/report/html_report/), but build still failed 😞 
Would be very grateful for the advice. [Screenshort](https://prnt.sc/tUGUW9_J8GPE)